### ISSUE TITLE
Elm v0.19

### ISSUE THREAD

**Issue by Bastes** [arturopala/elm-monocle#19]
With the removal of operators, I guess we're going to need some cleanup.

Would you like some help? If so, how do you think we should proceed? :)

**Comment by Bastes** [comment]
Sorry, you've already done it 😅 thanks.

**Comment by arturopala** [comment]
yeah, unfortunately, I did it only after releasing the new version, so elm
documentation site shows operator imports

śr., 29 sie 2018 o 10:32 użytkownik Michel Belleville <
notifications@github.com> napisał:
